openfisca_france/parameters/geopolitique/regions/liste.yaml

```
@@ -0,0 +1,21 @@
+description: Liste des régions administratives de France
+values:
+  2016-01-01:
+    value: ['guadeloupe',
+            'martinique',
+            'guyane',
+            'reunion',
+            'mayotte',
+            'ile_de_france',
+            'centre_val_de_loire',
+            'bourgogne_franche_comte',
+            'normandie',
+            'hauts_de_france',
+            'grand_est',
+            'pays_de_la_loire',
+            'bretagne',
+            'nouvelle_aquitaine',
+            'occitanie',
+            'auvergne_rhone_alpes',
+            'provence_alpes_cote_d_azur',
+            'corse']
```

```suggestion
    value:
      - guadeloupe
      - martinique
      - guyane
      - reunion
      - mayotte
      - ile_de_france
      - centre_val_de_loire
      - bourgogne_franche_comte
      - normandie
      - hauts_de_france
      - grand_est
      - pays_de_la_loire
      - bretagne
      - nouvelle_aquitaine
      - occitanie
      - auvergne_rhone_alpes
      - provence_alpes_cote_d_azur
      - corse
```

Il manque aussi un saut de ligne.